Gravity/Gravity.Test.Integration/App.config

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hey, I don't think we want to commit this. and make sure to change your password as this is public now.  FYI if you run git update-index --assume-unchanged Gravity/Gravity.Test.Integration/App.config when you pull or branch, this should stop this file from being tracked.